web/components/multi-user-transaction-link.tsx

```diff
@@ -48,27 +48,41 @@ export function MultiUserTransactionModal(props: {
   modalLabel: string
   open: boolean
   setOpen: (open: boolean) => void
+  short?: boolean
 }) {
-  const { userInfos, modalLabel, open, setOpen } = props
+  const { userInfos, modalLabel, open, setOpen, short } = props
+  const hasUsers = userInfos.length > 0
   return (
     <Modal open={open} setOpen={setOpen} size={'sm'}>
-      <Col className="items-start gap-4 rounded-md bg-white p-6">
-        <span className={'text-xl'}>{modalLabel}</span>
-        {userInfos.map((userInfo) => (
-          <Row
-            key={userInfo.username + 'list'}
-            className="w-full items-center gap-2"
-          >
-            <span className="min-w-[3.5rem] text-teal-500">
-              +{formatMoney(userInfo.amount)}
-            </span>
-            <Avatar
-              username={userInfo.username}
-              avatarUrl={userInfo.avatarUrl}
-            />
-            <UserLink name={userInfo.name} username={userInfo.username} />
-          </Row>
-        ))}
+      <Col className="relative items-start gap-4 rounded-md bg-white p-6">
+        <span className={'sticky top-0 text-xl'}>{modalLabel}</span>
+        {hasUsers && (
+          <Col className="max-h-96 w-full gap-4 overflow-y-scroll">
```

Comment:
should be `overflow-y-auto`

<img width="398" alt="image" src="https://user-images.githubusercontent.com/18368938/206761265-5e0913a9-cc43-47df-8acc-8f70b90e81b5.png">